src/backend/distributed/deparser/deparse_sequence_stmts.c

```diff
@@ -88,7 +89,8 @@ AppendSequenceNameList(StringInfo buf, List *objects, ObjectType objtype)
 
 		if (seq->schemaname == NULL)
```

Comment:
Generally we don't read from catalog tables during deparsing. Instead we simply deparse the statement as it was passed in. When the schemaname is not set, we simply don't deparse it.

Given this is already implemented by `quote_qualified_identifier` (skipping the qualifier if its a `NULL`) we can simply skip this whole if statement. Could you please remove it here?

Any filling of schema name needs to be solely done via qualifying the statement before deparsing. Lets unconvolute this section while we are passing by anyway.